official/README.md

```diff
@@ -22,3 +21,11 @@ Below is a list of the models available.
 More models to come!
 
 If you would like to make any fixes or improvements to the models, please [submit a pull request](https://github.com/tensorflow/models/compare).
+
+---
+
+Run the models.
+
+The *Official Models* is made as a Python module. To run the models directly, you need to add the top-level ***/models*** folder to the python path by command: ` export PYTHONPATH="$PYTHONPATH:/path/to/models" `
+
```

Comment:
A few more nits that I started to type out, then realized it was easier just to show the end result:

```
## Running the models

The *Official Models* are made available as a Python module. To run the models and associated scripts, add the top-level ***/models*** folder to the Python path with the command: `export PYTHONPATH="$PYTHONPATH:/path/to/models"`

To make Official Models easier to use, we are planning to create a pip installable Official Models package. This is being tracked in [#917](https://github.com/tensorflow/models/issues/917).
```